src/FormBuilder/resources/js/form-builder/src/settings/form-summary/page-slug.js

```diff
@@ -11,71 +11,69 @@ const {
 } = getWindowData();
 
 /**
+ * @unreleased pass page slug from parent component to support form title as initial slug.
  * @since 3.0.0
  * @see https://github.com/WordPress/gutenberg/blob/a8c5605f5dd077a601aefce6f58409f54d7d4447/packages/editor/src/components/post-slug/index.js
  */
-const PageSlugControl = () => {
-    const {
-        settings: {pageSlug},
-    } = useFormState();
+const PageSlugControl = ({pageSlug}) => {
```

Comment:
I noticed this little quirk where if you update the title and click publish - the url below does not reflect the actual slug until you refresh the page or switch modes.  It makes me think the slug is being memoized somewhere (probably in this file). I think we should clean this ux up.
<img width="334" alt="Screenshot 2023-10-23 at 5 36 20 PM" src="https://github.com/impress-org/givewp/assets/10138447/4d96a847-4f83-402d-b52f-431108a740fe">

Reply:
How would you feel about listening for the pageSlug updates? As the props updating however the component is not re-rendering. : 31dd6b7703011c474620edc86ef2c362150061aa bc9dd440a557a2d18d05504e7084beb4ecd443ba

Reply:
yeah that works 👍